pkg/debug/scripts/collect-info.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+#
+# This script is used to collect all info from the machine which
+# helps to debug and resolve EVE issues.
+#
+
+DATE=$(date -Is)
+INFO_DIR="eve-info-$DATE"
+TARBALL_FILE="/persist/$INFO_DIR.tar.gz"
+
+# Create temporary dir
+TMP_DIR=$(mktemp -d)
+DIR="$TMP_DIR/$INFO_DIR"
+mkdir -p "$DIR"
+
+# Install usefull stuff
+apk add dmidecode
+
+# Have to chroot, lsusb does not work from this container
+chroot /hostfs lsusb -vvv    > "$DIR/lsusb-vvv"
+chroot /hostfs lsusb -vvv -t > "$DIR/lsusb-vvv-t"
+
+dmesg         > "$DIR/dmesg"
+ps awux       > "$DIR/ps-awux"
```

Comment:
If you are not installing ps here, it's not guaranteed that you have the GNU version instead of busybox, so it's better to remove these flags...

Reply:
yes, but my point is that this ps doesn't consider any flag at all, so `ps flakeflags` will produce exactly the same output of `ps`. So it's better to keep only `ps > "$DIR/ps"` to avoid any confusion and/or feature issue (and keep it simple).

Reply:
True.